Integrations/integration-UrlScan.yml

```diff
@@ -0,0 +1,411 @@
+description: Urlscan.io reputation
+detaileddescription: |-
```

Comment:
Just write "This is a free service with an API key required. contact urlscan.io to obtain one.

Shouldn't be that detailed :)